packages/kernel-6.1/kernel-6.1.spec

```diff
@@ -139,6 +142,12 @@ Summary: Modules for the Linux kernel on bare metal
 %description modules-metal
 %{summary}.
 
+%package mkfs-confs
+Summary: mkfs confiigurations for different file systems
+
+%description mkfs-confs
```

Comment:
nit:
```suggestion
%package xfs-mkfs-confs
```

Reply:
Why would we change this? I see all other packages have a description there

Reply:
The diff wasn't intended to be the description, sorry about that confusion. I see the intention is to make this package generic. I'm fine with this but then we should probably call out that in a comment that this package is intended to hold future other configuration files as well, rather than specifically grabbing the package for a particular filesystem. My mental model was that we would only include these files for supported filesystems, so we would group them for each one as we add them, but this works as well.